specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/examples/SuccessfulDetectObjectsWithStream.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "Endpoint": "https://westus.api.cognitive.microsoft.com",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Image": "{binary}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "objects": [
+          {
+            "rectangle": {
+              "x": 0,
+              "y": 0,
+              "w": 50,
+              "h": 50
+            },
+            "object": "tree",
+            "confidence": 0.9,
+            "parent": {
+              "object": "plant",
+              "confidence": 0.95
+            }
+          }
+        ],
+        "requestId": "1ad0e45e-b7b4-4be3-8042-53be96103337",
+        "metadata": {
+          "width": 100,
+          "height": 100,
+          "format": "Jpeg"
```

Comment:
Just an idea, wouldn't `jpeg` or `JPEG` be better?

Reply:
I agree, but for all the of other services it's `Jpeg` and I think it's better to keep them consistent.

Reply:
Just curious, what services you are referring to? I searched before and didn't find anyother..

Reply:
[Analyze](https://github.com/Azure/azure-rest-api-specs/blob/78e8a4782397b62033139381169eb45bd87e1743/specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/examples/SuccessfulAnalyzeImageByDomainWithStream.json#L15)
[Tag](https://github.com/Azure/azure-rest-api-specs/blob/78e8a4782397b62033139381169eb45bd87e1743/specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/examples/SuccessfulTagImageWithUrl.json#L48)
[Describe](https://github.com/Azure/azure-rest-api-specs/blob/78e8a4782397b62033139381169eb45bd87e1743/specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/examples/SuccessfulDescribeImageWithUrl.json#L39)

Reply:
Oh you mean other `operations`, I thought you mean other `services`